pkgs/development/tools/ocaml/ocamlformat/generic.nix

```diff
@@ -28,9 +28,7 @@ buildDunePackage {
 
   minimumOCamlVersion = "4.08";
 
-  useDune2 = true;
-
-  strictDeps = true;
```

Comment:
What does strictDeps do exactly? I notice that we remove it a lot here.

Reply:
The reason we remove it is that it's the default in `buildDunePackage` already so it's redundant.

But it makes sure the dependencies are but leaking in to the wrong bucket. Native dependencies (executables) will not be mixed with libraries etc.